Leaflet.Graticule.js

```diff
@@ -419,7 +480,7 @@ L.LatLngGraticule = L.Layer.extend({
                     ctx.lineTo(rr.x-1, rr.y);
                     ctx.stroke();
                     if (self.options.showLabel && label) {
-                        var _yy = ll.y + (txtHeight/2)-2;
+                        var _yy = ll.y - (txtHeight/2) + 2;
```

Comment:
Why this change? Seems unrelated at first sight.

Reply:
This change is about labels placements.